src/main/java/org/broadinstitute/consent/http/models/dataset_registration_v1/DatasetRegistrationSchemaV1.java

```diff
@@ -0,0 +1,1582 @@
+    "alternativeDataSharingPlanExplanation",
+    "alternativeDataSharingPlanFileName",
+    "alternativeDataSharingPlanDataSubmitted",
+    "alternativeDataSharingPlanDataReleased",
+    "alternativeDataSharingPlanTargetDeliveryDate",
+    "alternativeDataSharingPlanTargetPublicReleaseDate",
+    "alternativeDataSharingPlanControlledOpenAccess",
+    "dataAccessCommitteeId",
+    "consentGroups"
+})
+@Generated("jsonschema2pojo")
+public class DatasetRegistrationSchemaV1 {
+
+    /**
+     * The study name
+     * (Required)
+     *
+     */
+    @JsonProperty("studyName")
+    @JsonPropertyDescription("The study name")
+    private String studyName;
+    /**
+     * The study type
+     * (Required)
+     *
+     */
+    @JsonProperty("studyType")
+    @JsonPropertyDescription("The study type")
+    private DatasetRegistrationSchemaV1.StudyType studyType;
+    /**
+     * Description of the study
+     * (Required)
+     *
+     */
+    @JsonProperty("studyDescription")
+    @JsonPropertyDescription("Description of the study")
+    private String studyDescription;
+    /**
+     * All data types that study encompasses
+     * (Required)
+     *
+     */
+    @JsonProperty("dataTypes")
+    @JsonPropertyDescription("All data types that study encompasses")
+    private List<String> dataTypes = new ArrayList<String>();
+    /**
+     * List of File Types
+     * (Required)
+     *
+     */
+    @JsonProperty("fileTypes")
+    @JsonPropertyDescription("List of File Types")
+    private List<FileTypeObject> fileTypes = new ArrayList<FileTypeObject>();
+    /**
+     * Phenotype/Indication Studied
+     * (Required)
+     *
+     */
+    @JsonProperty("phenotypeIndication")
+    @JsonPropertyDescription("Phenotype/Indication Studied")
+    private String phenotypeIndication;
+    /**
+     * Species
+     * (Required)
+     *
+     */
+    @JsonProperty("species")
+    @JsonPropertyDescription("Species")
+    private String species;
+    /**
+     * Principal Investigator Name
+     * (Required)
+     *
+     */
+    @JsonProperty("piName")
+    @JsonPropertyDescription("Principal Investigator Name")
+    private String piName;
+    /**
+     * The user creating the dataset submission
+     * (Required)
+     *
+     */
+    @JsonProperty("dataSubmitterUserId")
+    @JsonPropertyDescription("The user creating the dataset submission")
+    private Integer dataSubmitterUserId;
+    /**
+     * Data Custodian Email
+     * (Required)
+     *
+     */
+    @JsonProperty("dataCustodianEmail")
+    @JsonPropertyDescription("Data Custodian Email")
+    private List<String> dataCustodianEmail = new ArrayList<String>();
+    /**
+     * Public Visibility of this study
+     * (Required)
+     *
+     */
+    @JsonProperty("publicVisibility")
+    @JsonPropertyDescription("Public Visibility of this study")
+    private Boolean publicVisibility;
+    @JsonProperty("nihAnvilUse")
+    private List<NihAnvilUse> nihAnvilUse = new ArrayList<NihAnvilUse>();
```

Comment:
needs comment ?

Reply:
👍🏽 This is because I forgot to add a description in the schema for this field.